src/jdk.compiler/share/classes/com/sun/tools/javac/resources/launcher_zh_CN.properties

```diff
@@ -120,7 +120,7 @@ launcher.err.cant.read.file=读取源文件 {0} 时出错：{1}
 launcher.err.no.value.for.option=没有为选项 {0} 指定值
 
 # 0: string
-launcher.err.invalid.value.for.source=--source 选项的值无效：{0}\n
+launcher.err.invalid.value.for.source=--source 选项的值无效：{0}
```

Comment:
Looks correct. Removal of `\n` is done by white space tool, and standardizes this value with the original and other localized values that do not end with `\n`.